assignment.js

```diff
@@ -63,9 +76,29 @@ Use the following test cases to confirm your program meets the success criteria
 ************/
 
 console.log('Problem 2:')
-
+console.log('')
 // Add your code below this line
 
+const numbers = [57, 13, 4, 38]
+const numbers1 = [3, 48, 29, 12, 18]
+
+//finds highest number in the array
+let currentLargestNumber = numbers.reduce(function(a, b) {
+    return Math.max(a, b);
+});
+let currentLargestNumber1 = numbers1.reduce(function(a, b) {
+    return Math.max(a, b);
+});
+
+
+
+console.log(currentLargestNumber)
+console.log(currentLargestNumber1)
+
```

Comment:
@jct117 Good Job using the array method `reduce` and using the Math.max to compare each value to find the largest number. One suggestion take a look at your indentation on `line 87` and `line 90` you wanna make sure you indent with 2 spaces currently it has 4 spaces. 
Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L76)